From dec91454c2a5e72fccbd5e931a8bda39aeeb0edb Mon Sep 17 00:00:00 2001
Message-ID: <dec91454c2a5e72fccbd5e931a8bda39aeeb0edb.1732384228.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1732384228.git.sharlatanus@gmail.com>
References: <cover.1732384228.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 21 Nov 2024 19:32:43 +0000
Subject: [PATCH 42/46] gnu: python-stsynphot: Update to 1.4.0.

* gnu/packages/astronomy.scm (python-stsynphot): Update to 1.4.0.
[native-inputs]: Add python-setuptools and python-wheel.

Change-Id: I9155f517358ecf651a47fe48a676202a2c19bb2f
---
 gnu/packages/astronomy.scm | 38 +++++++++++++++++++++-----------------
 1 file changed, 21 insertions(+), 17 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 82275ebbda..1295b04f6b 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -5263,13 +5263,13 @@ (define-public python-stpipe
 (define-public python-stsynphot
   (package
     (name "python-stsynphot")
-    (version "1.3.0")
-    (source (origin
-              (method url-fetch)
-              (uri (pypi-uri "stsynphot" version))
-              (sha256
-               (base32
-                "15m3vy39mc7rh1mrxs8d2bs0bx5vv632m6wg8n15fqjzhadqy7j5"))))
+    (version "1.4.0")
+    (source
+     (origin
+       (method url-fetch)
+       (uri (pypi-uri "stsynphot" version))
+       (sha256
+        (base32 "02xaglg1kv3mk6gqjcs1283mmy9b42fgsga8g2z8768lkgwmmg9j"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -5278,16 +5278,20 @@ (define-public python-stsynphot
       ;; Failed to load Vega spectrum from
       ;; /grp/redcat/trds/calspec/alpha_lyr_stis_010.fits;
       #:tests? #f))
-    (propagated-inputs (list python-astropy
-                             python-beautifulsoup4
-                             python-matplotlib
-                             python-numpy
-                             python-scipy
-                             python-synphot))
-    (native-inputs (list python-pytest
-                         python-pytest-astropy
-                         python-pytest-astropy-header
-                         python-setuptools-scm))
+    (native-inputs
+     (list python-pytest
+           python-pytest-astropy
+           python-pytest-astropy-header
+           python-setuptools
+           python-setuptools-scm
+           python-wheel))
+    (propagated-inputs
+     (list python-astropy
+           python-beautifulsoup4
+           python-matplotlib
+           python-numpy
+           python-scipy
+           python-synphot))
     (home-page "https://github.com/spacetelescope/stsynphot_refactor")
     (synopsis "Synthetic photometry using Astropy for HST and JWST")
     (description
-- 
2.46.0

